internal/acceptance/openstack/objectstorage/v1/objects_test.go

```diff
@@ -416,7 +415,6 @@ func TestObjectsBulkDelete(t *testing.T) {
 
 	// Verify deletion
 	listOpts := objects.ListOpts{
-		Full:      true,
```

Comment:
List option `Full: false` (`accept: text/plain`) doesn't work and should not be used. We always list "full" now (`accept: application/json`).